packages/framework/src/Model/Order/OrderFacade.php

```diff
@@ -503,7 +503,7 @@ protected function fillOrderItems(Order $order, OrderPreview $orderPreview)
         $this->fillOrderProducts($order, $orderPreview, $locale);
         $this->fillOrderPayment($order, $orderPreview, $locale);
         $this->fillOrderTransport($order, $orderPreview, $locale);
-        $order->fillOrderRounding($this->orderItemFactory, $orderPreview->getRoundingPrice(), $locale);
+        $this->fillOrderRounding($order, $orderPreview, $locale);
```

Comment:
so this is because of some pattern of methods above?

Reply:
I do not understand your question :confused: